roles/icvpn/tasks/bird.yml

```diff
@@ -14,5 +14,13 @@
 - name: Install cronjob to automatically update bgp bird configurations
   template:
     src: icvpn-update
-    dest: /etc/cron.hourly/icvpn-update
+    dest: /usr/local/bin/icvpn-update
     mode: 0755
+
+- name: Install timer
+  import_role:
+    name: timer
+  vars:
+    timer_name: icvpn-update
+    timer_exec: /usr/local/bin/icvpn-update
+    timer_interval: 1hour
```

Comment:
missing remove of `/etc/cron.hourly/icvpn-update`

Reply:
done